.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+## Purpose
+_Describe the problem or feature in addition to a link to the issues._
+
+## Approach
```

Comment:
on the more pragmatic side, we should add

## Testing
_Describe how the changes were tested._

Reply:
I like how you think sir — added.